README.md

```diff
@@ -216,6 +216,23 @@ You're probably thinking something like: "this is a large amount of work just to
 1.  Forces decoupling. In our example, this is illustrated by decoupling our configuration and database configuration.
 2.  After a type is configured, it can be injected anywhere with no additional effort. Simply `@inject` and it appears. We don't really illustrate that here, but you can imagine adding an arbitrary number of `RequestHandler` subclasses, all of which will automatically have a DB connection provided.
 
+
+First time setup in development mode
+------------------------------------
+
+Install `Injector` in editable mode with development dependencies:
+
+```bash
+pip install -e '.[dev]'
+```
+
+Running the tests:
+
+```bash
+pytest
+```
```

Comment:
Let's mention `python setup.py test` as an alternative as well.

Additionally I believe we need to mention `mypy` and `black` as well, since they're called from the CI pipeline.

Reply:
>Let's mention python setup.py test as an alternative as well.

It's redundancy because
1. nobody uses it besides you
2. It's not flexible as pytest cause developers can't pass arguments with a test name for example.

I prefer to have one perfect way of doing something instead of a couple of so-so.
You should too. 😉

>Additionally I believe we need to mention mypy and black as well, since they're called from the CI pipeline.

We should mention them only if we listed them in requirements-dev otherwise it's pointless.

Reply:
It's a redundancy I can live with. `python setup.py test` is a standard way to run the test suite and I can easily imagine people integrating this library run this. I know well of the limitations of it but I want it to stay since it's already implemented. Please mention it.

`black` and `mypy` *do* need to be mentioned in `requirements-dev.txt` so this makes the situation clear (even if they didn't the documentation should still mention them, documenting just `pytest` is incomplete and misleading).

Reply:
>It's a redundancy I can live with.
>Please mention it.

Sounds more like a religion or just a habit. OK.

>black and mypy do need to be mentioned

Fair enough.

Reply:
> Sounds more like a religion or just a habit

Please refrain from making comments like this one, telling a maintainer of a project what they should be doing or that how they're doing things is wrong (other comments). Especially when you probably don't fully know the maintainer's motivation. This is not helpful and will lead nowhere.